src/geocoder/geocoder.ts

```diff
@@ -672,7 +669,9 @@ module powerbi.extensibility.geocoder {
                 return;
             }
 
-            GeocodeCallback = (data) => {
+            let guid =  "GeocodeCallback" + new Date().getTime() + (((1+Math.random())*0x10000)|0).toString(16).substring(1);;
```

Comment:
I created some test for you to ensure that this method is ok to use - please check it here: https://jsfiddle.net/jw7pd2d0/1/